internal/docker/container.go

```diff
@@ -20,12 +20,31 @@ import (
 // ContainerRunner represents the container runner for docker.
 type ContainerRunner struct {
 	Ctx             context.Context
-	Cli             *command.SauceCtlCli
-	containerID     string
 	docker          *Handler
 	containerConfig *containerConfig
 	Framework       framework.Framework
 	ImageLoc        framework.ImageLocator
+	ShowConsoleLog  bool
+}
+
+// containerStartOptions represent data required to start a new container.
+type containerStartOptions struct {
+	Docker      config.Docker
+	BeforeExec  []string
+	Project     interface{}
+	SuiteName   string
+	Environment map[string]string
+	Files       []string
+}
+
+// result represents the result of a local job
+type result struct {
+	containerID   string
+	err           error
+	passed        bool
+	output        string
```

Comment:
Imho it's not very clear what `output` refers to. Luckily it's just a local struct, so other packages don't need to care, but maybe there is a bit or a more descriptive name we can find for this? 🤔

Reply:
@alexplischke would `consoleOutput` be better ?

Reply:
If that's what it contains, then absolutely! 👍